packages/java-edition/src/mcfunction/parser/argument.ts

```diff
@@ -1640,3 +1731,173 @@ function vector(
 		return ans
 	}
 }
+
+function components(): core.InfallibleParser<ComponentListNode> {
+	return core.map(
+		core.record({
+			start: '[',
+			pair: {
+				// TODO: Create an item_component category
+				key: core.resourceLocation({
+					pool: [],
+					allowUnknown: true,
+					namespacePathSep: '.',
```

Comment:
Why is `namespacePathSep` set to `.`? I think the game uses `:` to separate namespace and path here.